xml/ha_storage_protection.xml

```diff
@@ -875,9 +875,11 @@ Received command test from &node2; on disk /dev/disk/by-id/<replaceable>DEVICE_I
       and survives in a split-brain scenario:
       </para>
 <screen>&prompt.crm.conf;<command>primitive st-sbd-&node1; stonith:external/sbd params \
-pcmk_host_list=&node1; pcmk_delay_base=20</command>
+  pcmk_host_list=&node1; pcmk_delay_base=20 \
+  op monitor timeout=20s interval=3600s</command>
```

Comment:
Again, these "sbd" related changes are not needed. There was the long debate in the past. In general, we don't add "monitor" for all fencing devices in CIB.